pecos/xmc/xlinear/model.py

```diff
@@ -359,13 +359,15 @@ def predict(
         self,
         X,
         pred_params=None,
+        select_outputs_csr=None,
         **kwargs,
     ):
         """Predict on given input data
 
         Args:
             X (csr_matrix(float32) or ndarray(float32)): instance feature matrix to predict on
             pred_params (XLinearModel.PredParams, optional): instance of XLinearModel.PredParams
+            select_outputs_csr (csr_matrix, optional): instance label matrix to predict from
```

Comment:
Add expected matrix shape and make it clear only its indices will be used but not the value.

Reply:
Docstring modified to specify shape of select_outputs_csr. Note has been added to make it clear that only indices with nonzero values will be predicted.